tests/function-controller/go.mod

```diff
@@ -10,24 +10,29 @@ require (
 	github.com/hashicorp/go-multierror v1.1.0
 	github.com/kubernetes-sigs/service-catalog v0.3.0
 	github.com/kyma-project/helm-broker v1.1.0
+	github.com/kyma-project/kyma/components/eventing-controller v0.0.0-20210311150052-653b9d72e0d7
 	github.com/kyma-project/kyma/components/function-controller v0.0.0-20201201100156-840f2b1a3da8
 	github.com/onsi/gomega v1.10.2
 	github.com/pkg/errors v0.9.1
 	github.com/sirupsen/logrus v1.6.0
 	github.com/spf13/pflag v1.0.5
 	github.com/vrischmann/envconfig v1.3.0
-	golang.org/x/crypto v0.0.0-20200820211705-5c72a883971a // indirect
 	golang.org/x/sync v0.0.0-20200625203802-6e8e738ad208
 	honnef.co/go/tools v0.0.1-2020.1.5 // indirect
-	k8s.io/api v0.18.12
-	k8s.io/apimachinery v0.18.12
+	k8s.io/api v0.19.3
+	k8s.io/apimachinery v0.19.3
 	k8s.io/client-go v11.0.1-0.20190805182717-6502b5e7b1b5+incompatible
 	sigs.k8s.io/controller-runtime v0.6.4
 )
 
 replace (
+	github.com/googleapis/gnostic => github.com/googleapis/gnostic v0.3.1
+	github.com/kyma-project/kyma => github.com/kyma-project/kyma v0.5.1-0.20210302111801-5a4f7610cf45
```

Comment:
Do we need this?

Reply:
I think it is needed as the latest gnostic was not compliant with the rest.

Reply:
I mean `kyma-project`